spec/controllers/pages_controller_spec.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe PagesController, type: :controller do
```

Comment:
It would be good to write this test. If you don't want to do this add new task in trello for this issue. 

Generally we should concentrate on tests in this project.